mazda_cx5_gt_2017.dbc

```diff
@@ -491,8 +491,10 @@ BO_ 552 GEAR: 8 XXX
  SG_ NEW_SIGNAL_5 : 26|3@0+ (1,0) [0|255] "" XXX
  SG_ NEW_SIGNAL_6 : 31|5@0+ (1,0) [0|31] "" XXX
  SG_ NEW_SIGNAL_7 : 39|1@0+ (1,0) [0|255] "" XXX
- SG_ GEAR : 35|4@0+ (1,0) [0|15] "" XXX
  SG_ MORE_GEAR : 7|4@0+ (1,0) [0|255] "" XXX
+ SG_ GEAR : 35|3@0+ (1,0) [0|15] "" XXX
+ SG_ NEW_SIGNAL_1 : 0|3@1+ (1,0) [0|3] "" XXX
```

Comment:
don't need to define unknown signals

Reply:
want me do drop that bit ?

Reply:
yeah or give it a better name if you know what it is

Reply:
coming your way... with GEAR values updated to allow parsing. Will wait for this PR to trickle in OP before updating the Mazda port PR for anything that depend on it